packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/reducers.ts

```diff
@@ -15,7 +15,8 @@ const INITIAL_STATE: BrokerageState = {
   account: undefined,
   redirectBackToStep: false,
   addNew: false, // TODO: Put this stuff in redux-form
-  bankStatus: Remote.NotAsked
+  bankStatus: Remote.NotAsked,
+  fiatCurrency: undefined
```

Comment:
Let's make a not that when we pull out cards from simple buy into brokerage we can reuse this since there is a `fiatCurrency` defined in the simplebuy state as well.